jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListViewImpl.java

```diff
@@ -187,9 +193,17 @@ public void initColumns(final ListTable<ProcessInstanceSummary> extendedPagedTab
         extendedPagedTable.getColumnSortList().push(startColumn);
     }
 
-    public void addDomainSpecifColumns(ExtendedPagedTable<ProcessInstanceSummary> extendedPagedTable,
-                                       Set<String> columns) {
+    protected boolean isSameColumn(GridColumnPreference gridColumnPreference, List<ColumnMeta<ProcessInstanceSummary>> columns) {
+        for (ColumnMeta<ProcessInstanceSummary> columnMeta : columns) {
+            if (columnMeta.getCaption().equals(gridColumnPreference.getName())) {
+                return true;
+            }
+        }
+        return false;
+    }
 
+    public void addDomainSpecifColumns(ExtendedPagedTable<ProcessInstanceSummary> extendedPagedTable,
```

Comment:
could some of the code be shared between this and the modified method above?

Reply:
I will take a try.